csharp/Yoga/CMakeLists.txt

```
@@ -0,0 +1,69 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.4.1)
```

```suggestion
cmake_minimum_required(VERSION 3.14)
```

Planning to bump the min-version across the board in the main CMake build so that we can use `target_link_options` and newer policies.